src/Spago/Command/Publish.purs

```diff
@@ -125,9 +126,12 @@ publish _args = do
     )
 
   -- We then need to check that the dependency graph is accurate. If not, queue the errors
-  let globs = Build.getBuildGlobs { selected: [ selected ], withTests: false, dependencies, depsOnly: false }
-  graphCheckErrors <- Graph.runGraphCheck selected globs []
-  for_ graphCheckErrors addError
+  let dependencies = Fetch.getAllDependencies packageDependencies
```

Comment:
Looking at this diff reminds me that we need a dry-run kind of test for the publish command, there is nothing that ensures that it keeps working at the moment 😅

Reply:
So, is this you saying that we need to add a test for that inside or outside this PR? I'm hoping outside?

Reply:
Of course, definitely not here! I was just issuing a general reminder for both of us 😄